README.md

```diff
@@ -173,7 +173,17 @@ OAuth will work.
                "useLetsEnc": true,
                "redirect": true,
                "upstreamURL": ""
+         },
+         {
+               "domain": "<custom-actions-subdomain>.<custom-domain>",
```

Comment:
Should this say `custom-playout-subdomain`?

It would also be good to add the realtime service to this list, which would be `custom-realtime-subdomain` and port `3002`